packages/applicant/src/services/GrantAdvertService.tsx

```diff
@@ -16,6 +16,17 @@ export async function getAdvertBySlug(
   return data;
 }
 
+export async function getAdvertBySchemeId(
+  schemeId: string,
+  jwt: string
+): Promise<AdvertDto> {
```

Comment:
Type can be inferred
```suggestion
export async function getAdvertBySchemeId(
  schemeId: string,
  jwt: string
) {
```